### ISSUE TITLE
Initialising ZIO runtime freezes the language server

### ISSUE THREAD

**Issue by PabloBuchu** [enso-org/enso#6028]
Language engines freezes during initialisation on `Initializing ZIO runtime...`. This blocks any connections from success: json, binary or even call for idle time `curl http://runtime:30001/_idle`.

This regression block cloud from starting and using new projects

discord thread https://discord.com/channels/401396655599124480/1087703965648560248

logs file:
[message (3).txt](https://github.com/enso-org/enso/files/11028635/message.3.txt)

**Comment by 4e6** [comment]
The thread dump shows that the call to initialize ZIO indeed hangs

```
java.lang.Thread.State: WAITING (on object monitor)
    at java.lang.Object.wait(java.base@11.0.18/Native Method)
    - waiting on <0x00000007596fe878> (a zio.internal.OneShot)
    at java.lang.Object.wait(java.base@11.0.18/Object.java:328)
    at zio.internal.OneShot.get(OneShot.scala:79)
    - waiting to re-lock in wait() <0x00000007596fe878> (a zio.internal.OneShot)
    at zio.Runtime.unsafeRunSync(Runtime.scala:85)
    at zio.Runtime.unsafeRunSync$(Runtime.scala:80)
    at zio.Runtime$$anon$3.unsafeRunSync(Runtime.scala:280)
    at zio.Runtime.unsafeRun(Runtime.scala:58)
    at zio.Runtime.unsafeRun$(Runtime.scala:57)
    at zio.Runtime$$anon$3.unsafeRun(Runtime.scala:280)
    at zio.Runtime$.unsafeFromLayer(Runtime.scala:304)
    at org.enso.languageserver.effect.Runtime$.fromExecutionContext(Runtime.scala:27)
    at org.enso.languageserver.effect.ExecutionContextRuntime.init(Runtime.scala:42)
```

[message.txt](https://github.com/enso-org/enso/files/11028847/message.1.txt)

**Comment by 4e6** [comment]
Fixed in #6069
